src/pages/home/report/ReportActionsList.tsx

```diff
@@ -329,6 +330,19 @@ function ReportActionsList({
         };
     }, [report.reportID]);
 
+    useEffect(() => {
+        // Reset yellow highlight when user refreshed the page
+        if (!linkedReportActionID || getPlatform() !== 'web') {
+            return;
+        }
+        const navigationEntries = performance.getEntriesByType('navigation');
+        if (!navigationEntries.length || (navigationEntries[0] as PerformanceNavigationTiming).type !== 'reload') {
+            return;
+        }
+        Navigation.setParams({reportActionID: ''});
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [linkedReportActionID]);
+
```

Comment:
@sobitneupane I tested some alternatives and didn't find a better way to **identify when the user refreshed the page** without breaking the "Comment Linking" feat. Can you think of a better way?

Reply:
@sobitneupane I think found a bug, not related to this PR, however making it not possible to test correctly:
Only in `native`, when we return to the main chat, the yellow highlight appears as expected, but the new message button appears no matter the size of the conversation. Even if the user takes action, the scroll doesn't go down, and the yellow highlight doesn't disappear. I did a quick search and didn't find a related issue, I'll look further.

Reply:
@brunovjk Can you please share the screen recording of the issue. 

The issue is "We don't remove the yellow highlight and scroll  doesn't go down even after user takes action in native devices", right? Is [this](https://github.com/Expensify/App/pull/43581#discussion_r1645019146) the same issue?

Reply:
**Reproduced in the last main 06/20**:
<details>
<summary>When returning from a theard, the "New Message" flag is active even when we have few messages, and does not disappear when scrolling.</summary>

<!-- add screenshots or videos here -->

https://github.com/Expensify/App/assets/95647348/57da200e-3f1f-4a5f-bc14-0c5c42c28c8a

</details>

Steps to reproduce:
1. Login to the native app (iOS or Android)
2. Go to any chat, click on "Reply on Theard" in the last message of that chat.
3. Click back into the main conversation by clicking the the room title in the top left of the thread window
4. Thread is highlighted in yellow after you exit it
5. Note that the "New Message" flag is active even when we have few messages, and does not disappear when scrolling. Try send a new message, it also doesn't desappear and it doesn't scroll to the end. On other platforms it works normally.

Reply:
@brunovjk Has this been resolved?

Reply:
<details>
<summary>Not yet, I haven't found any related issue, I'll report it.</summary>
<!-- add screenshots or videos here -->

https://github.com/Expensify/App/assets/95647348/ffcd1e0d-f0dc-4e02-a15d-cb3ee760ff42

</details>

Anyway, I'm still not 100% confident how Im dealing with "checking if the user refreshed the page". I'm working on it at the moment.

Reply:
> @brunovjk Has this been resolved?

@ sobitneupane The issue has been resolved. I'll keep testing :D

Reply:
Great. Please let me know when the PR is ready for review.